cis194/week8/tien/app/Main.hs

```diff
@@ -1,4 +1,14 @@
 module Main where
 
+import Party
+import System.Directory
+import System.IO
+
 main :: IO ()
-main = putStrLn "no use"
+main = do
+  dir <- getCurrentDirectory
+  handle <- openFile (dir ++ "/src/company.txt") ReadMode
+  contents <- hGetContents handle
+  putStrLn $ "Total Fun: " ++ (show . totalFun . read $ contents)
+  mapM_ putStrLn . totalEmpNameList . read $ contents
```

Comment:
`read` is partial. We can add a simple error handling if company.txt is malformed.